src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,89 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
+        System.out.println("Make your choise: ");
+        System.out.println("Enter 'add' to perform addition.");
+        System.out.println("Enter 'sub' to perform subtraction.");
+        System.out.println("Enter 'mul' to perform multiplication.");
+        System.out.println("Enter 'div' to perform division.");
+        System.out.println("Enter 'exit' to exit.");
+        System.out.println("Enter 'help' to see help message.");
+        boolean eq = true;
+        while (eq) {
```

Comment:
Please see this sollution, it's a little bit better) 
http://stackoverflow.com/questions/22823395/java-how-can-i-break-a-while-loop-under-a-switch-statement